pkg/clusterrouter/proxy/proxy_server.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/rancher/rancher/pkg/types/config"
 	"github.com/rancher/rancher/pkg/types/config/dialer"
 	"github.com/rancher/wrangler/pkg/schemas/validation"
+	"github.com/sirupsen/logrus"
```

Comment:
Not seeing usages of `logrus` here. Leftover from when debug logging was present?

Reply:
Yep, fixed